--- driver/Makefile.org	2003-08-03 18:50:09.000000000 +0000
+++ driver/Makefile	2003-10-19 11:35:49.000000000 +0000
@@ -17,7 +17,7 @@
 CC=gcc
 CFLAGS=-O2 -D__KERNEL__ -DMODULE -Wall -g -c $(EXTRA_CFLAGS)
 
-include $(KERNEL_PATH)/.config
+include kernel-config
 
 INCLUDES=-I$(KERNEL_PATH)/include
 ifdef PCMCIA_PATH
@@ -198,14 +198,14 @@
 endif
 
 install_check:
-	@if [ ! -d "$(MODPATH)" ]; then \
-		echo "Module installation directory not valid: '$(MODPATH)'"; \
-		echo -n "Most probable cause for this is modified kernel "; \
-		echo "headers."; \
-		echo -n "You will need to install module file "; \
-		echo "($(MSRC)/hostap*.{,k}o) manually."; \
-		exit 1; \
-	fi
+#	@if [ ! -d "$(MODPATH)" ]; then \
+#		echo "Module installation directory not valid: '$(MODPATH)'"; \
+#		echo -n "Most probable cause for this is modified kernel "; \
+#		echo "headers."; \
+#		echo -n "You will need to install module file "; \
+#		echo "($(MSRC)/hostap*.{,k}o) manually."; \
+#		exit 1; \
+#	fi
 
 install_2.4: uninstall install_pccard install_pci install_plx
 
@@ -217,12 +217,6 @@
 	@if [ -r $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko ]; then \
 		echo "You may need to remove old $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko"; \
 	fi
-	@if [ -f "/boot/System.map-$(KERNELRELEASE)" ] || [ "$(CURRENTKV)" = "$(KERNELRELEASE)" ]; then \
-		echo "*** Running depmod..."; \
-		/sbin/depmod -ae; \
-	else \
-		echo "*** I won't run depmod because I can't find an appropriate System.map file or you are not running the same kernel as the one you are building against. Please run it manually when the conditions are correct..."; \
-	fi
 
 install_hostap: $(MSRC)/hostap.o
 	@echo "Installing hostap.o to $(MODPATH_PCI)"
@@ -241,30 +235,25 @@
 	@echo "Installing $(MODULE_CS) to $(MODPATH_CS)"
 	mkdir -p $(MODPATH_CS)
 	cp -f $(MSRC)/$(MODULE_CS) $(MODPATH_CS)
-ifndef DESTDIR
-	/sbin/depmod -ae
-endif
 
 install_conf:
-	@if [ ! -r $(DESTDIR)/etc/pcmcia/hostap_cs.conf ]; then \
-		echo "Installing $(DESTDIR)/etc/pcmcia/hostap_cs.conf"; \
-		cp -f driver/etc/hostap_cs.conf $(DESTDIR)/etc/pcmcia/hostap_cs.conf; \
-	fi
-	@if [ -r $(DESTDIR)/etc/pcmcia/prism2.conf ]; then \
-		echo "You may need to remove old $(DESTDIR)/etc/pcmcia/prism2.conf"; \
-	fi
+#	@if [ ! -r $(DESTDIR)/etc/pcmcia/hostap_cs.conf ]; then \
+#		echo "Installing $(DESTDIR)/etc/pcmcia/hostap_cs.conf"; \
+#		cp -f driver/etc/hostap_cs.conf $(DESTDIR)/etc/pcmcia/hostap_cs.conf; \
+#	fi
+#	@if [ -r $(DESTDIR)/etc/pcmcia/prism2.conf ]; then \
+#		echo "You may need to remove old $(DESTDIR)/etc/pcmcia/prism2.conf"; \
+#	fi
 
 install_plx: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PLX) to $(MODPATH_PLX)"
 	mkdir -p $(MODPATH_PLX)
 	cp -f $(MSRC)/$(MODULE_PLX) $(MODPATH_PLX)
-	/sbin/depmod -ae
 
 install_pci: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PCI) to $(MODPATH_PCI)"
 	mkdir -p $(MODPATH_PCI)
 	cp -f $(MSRC)/$(MODULE_PCI) $(MODPATH_PCI)
-	/sbin/depmod -ae
 
 uninstall:
 	find $(MODPATH) -name "hostap*" | xargs rm -f
