--- Makefile.org	2004-05-24 00:48:05.000000000 +0000
+++ Makefile	2004-06-05 05:34:27.880662840 +0000
@@ -21,7 +21,7 @@ DESTDIR=
 CC=gcc
 CFLAGS=-O2 -D__KERNEL__ -DMODULE -Wall -g -c $(EXTRA_CFLAGS)
 
-include $(O)/.config
+include kernel-config
 
 INCLUDES=-I$(KERNEL_PATH)/include
 ifdef PCMCIA_PATH
@@ -146,12 +146,12 @@ install_2.6: install_conf
 	@if [ -r $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko ]; then \
 		echo "You may need to remove old $(MODPATH)/kernel/drivers/net/wireless/hostap_crypt.ko"; \
 	fi
-	@if [ -f "/boot/System.map-$(KERNELRELEASE)" ] || [ "$(CURRENTKV)" = "$(KERNELRELEASE)" ]; then \
-		echo "*** Running depmod..."; \
-		/sbin/depmod -ae; \
-	else \
-		echo "*** I won't run depmod because I can't find an appropriate System.map file or you are not running the same kernel as the one you are building against. Please run it manually when the conditions are correct..."; \
-	fi
+#	@if [ -f "/boot/System.map-$(KERNELRELEASE)" ] || [ "$(CURRENTKV)" = "$(KERNELRELEASE)" ]; then \
+#		echo "*** Running depmod..."; \
+#		/sbin/depmod -ae; \
+#	else \
+#		echo "*** I won't run depmod because I can't find an appropriate System.map file or you are not running the same kernel as the one you are building against. Please run it manually when the conditions are correct..."; \
+#	fi
 
 install_hostap: $(MSRC)/hostap.o
 	@echo "Installing hostap.o to $(MODPATH_PCI)"
@@ -170,30 +170,30 @@ install_pccard: install_check install_cr
 	@echo "Installing $(MODULE_CS) to $(MODPATH_CS)"
 	mkdir -p $(MODPATH_CS)
 	cp -f $(MSRC)/$(MODULE_CS) $(MODPATH_CS)
-ifndef DESTDIR
-	/sbin/depmod -ae
-endif
+#ifndef DESTDIR
+#	/sbin/depmod -ae
+#endif
 
 install_conf:
-	@if [ -d $(DESTDIR)/etc/pcmcia -a ! -r $(DESTDIR)/etc/pcmcia/hostap_cs.conf ]; then \
-		echo "Installing $(DESTDIR)/etc/pcmcia/hostap_cs.conf"; \
-		cp -f driver/etc/hostap_cs.conf $(DESTDIR)/etc/pcmcia/hostap_cs.conf; \
-	fi
-	@if [ -r $(DESTDIR)/etc/pcmcia/prism2.conf ]; then \
-		echo "You may need to remove old $(DESTDIR)/etc/pcmcia/prism2.conf"; \
-	fi
+#	@if [ -d $(DESTDIR)/etc/pcmcia -a ! -r $(DESTDIR)/etc/pcmcia/hostap_cs.conf ]; then \
+#		echo "Installing $(DESTDIR)/etc/pcmcia/hostap_cs.conf"; \
+#		cp -f driver/etc/hostap_cs.conf $(DESTDIR)/etc/pcmcia/hostap_cs.conf; \
+#	fi
+#	@if [ -r $(DESTDIR)/etc/pcmcia/prism2.conf ]; then \
+#		echo "You may need to remove old $(DESTDIR)/etc/pcmcia/prism2.conf"; \
+#	fi
 
 install_plx: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PLX) to $(MODPATH_PLX)"
 	mkdir -p $(MODPATH_PLX)
 	cp -f $(MSRC)/$(MODULE_PLX) $(MODPATH_PLX)
-	/sbin/depmod -ae
+#	/sbin/depmod -ae
 
 install_pci: install_check install_crypt install_hostap
 	@echo "Installing $(MODULE_PCI) to $(MODPATH_PCI)"
 	mkdir -p $(MODPATH_PCI)
 	cp -f $(MSRC)/$(MODULE_PCI) $(MODPATH_PCI)
-	/sbin/depmod -ae
+#	/sbin/depmod -ae
 
 uninstall:
 	find $(MODPATH) -name "hostap*" | xargs rm -f
